packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagelist/_mixins.scss

```diff
@@ -1,10 +1,24 @@
 @use "sass:math";
 
-@mixin chat-messagelist($padding) {
+@mixin chat-messagelist(
+  $padding,
+  $day-header-color,
+  $day-header-padding-bottom,
+  $first-day-header-padding-top,
+) {
   .dx-chat-messagelist {
     .dx-scrollable-content {
       padding-inline: $padding;
     }
+
+    .dx-chat-messagelist-day-header {
```

Comment:
Let's move the styles for .dx-chat-messagelist-day-header out of the .dx-chat-messagelist block. Also, let's use nested notation, as it improves readability

```
.dx-chat-messagelist-day-header {
    &:first-child {
    
    }
}
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925